xee/ext.py

```diff
@@ -195,6 +204,10 @@ def __init__(
       executor_kwargs = {}
     self.executor_kwargs = executor_kwargs
 
+    # Default value: (https://github.com/pydata/xarray/blob/main/xarray/backends/common.py#L181).
+    self.tile_fetch_max_retries = tile_fetch_kwargs.pop('max_retries', 6)
```

Comment:
Here we can do something like this ...

```
if tile_fetch_kwargs is None:
    self.tile_fetch_kwargs = TILE_FETCH_KWARGS

self.tile_fetch_kwargs = tile_fetch_kwargs
```

Reply:
This won't returns the appropriate output when the user only passed the single argument in the dict like `tile_fetch_kwargs = {'max_retries': 3}`.

P.S. I already update the code which works in the above case, too.

Reply:
But that's OK as we handle that below.